src/Administration/Resources/app/administration/src/module/sw-cms/page/sw-cms-list/index.js

```diff
@@ -580,23 +580,14 @@ export default {
             return isDefault ? `${defaultText} - ${typeLabel}` : typeLabel;
         },
 
-        /**
-         * @deprecated tag:v6.7.0 - Will be removed
-         */
         getPageCategoryCount(page) {
             return page.categories.length;
         },
```

Comment:
Just curiosity: Why no remove? Do we still need this?

Reply:
yes, still required and used